e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -1170,3 +1191,31 @@ function expectSearchResultContent({ expectedSearchResults, strict = true }) {
     );
   }
 }
+
+function createModelIndex({ modelId, pkName, valueName }) {
```

Comment:
instead of copying this from the model-indexes tests, let's extract it to a common helper file and import it in both places.